styles.css

```diff
@@ -1,3 +1,793 @@
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+
+.page-header {
+    display: flex;
+    align-items: center;
+    width: 1920px;
+    height: 108px;
+    border: 1px solid #000;
+    box-sizing: border-box;
+    padding: 0 100px;
+}
+
+.main-logo {
+    display: flex;
+    align-items: center;
+}
+
+.top-nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+}
+
+.nav-list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    text-decoration: none;
+    padding: 0;
+    width: 709px;
+    height: 24px;
+    gap: 32px;
+
+}
+
+.nav-item {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+}
+
+.nav-item .dropbtn {
+    background: none;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+}
+
+
+.nav-buttons {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 384px;
+    height: 48px;
+    gap: 20px;
+}
+
+.breadcrumbs-container {
+    background-image: url(images/breadcrumbs-img.svg);
+    background-color: #141B22;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+    height: 242px;
+    text-align: center;
+}
+
+.breadcrumbs {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 10px;
+}
+
+.breadcrumbs-nav {
+    display: flex;
+    align-items: center;
+}
+
+.breadcrumbs-nav a {
+    text-decoration: none;
+    color: #888B8E;
+    margin: 0 10px;
+}
+
+.about-us-section {
+    box-sizing: border-box;
+    display: inline-block;
+    justify-content: center;
+    width: 1410px;
+    height: 612px;
+    margin: auto;
+}
+
+.list-term+.list-description {
+    display: inline-block;
+}
+
+.section-text {
+    width: 930px;
+    margin: auto;
+}
+
+.ot-portrait {
+    border-radius: 50%;
+
+}
+
+.list-item {
+    display: inline-block;
+    width: 250px;
+    height: 150px;
+    margin: 0 5px;
+    background: #141B22;
+    padding: 10px;
+}
+
+.out-team {
+    width: 1410px;
+    height: 824px;
+    display: inline-block;
+    align-items: center;
+
+}
+
+.ot-members {
+    padding: 0;
+}
+
+.ot-member {
+    display: inline-block;
+    justify-content: center;
+    width: 330px;
+    height: 339px;
+    background: #141B22;
+    margin: 7px;
+}
+
+.ot-portrait {
+    border-radius: 50%;
+}
+
+.partners {
+    width: 1410px;
+    height: 474px;
+    margin: auto;
+}
+
+.join-our-community {
+    width: 1410px;
+    height: 246px;
+    border-radius: 8px 0px 0px 0px;
+    margin: auto;
+    background: linear-gradient(268.28deg, #2333DE -1.34%, #0ABE66 100%);
+}
+
+.partner-logo {
+    display: inline-block;
+    margin: 0 5px;
+    padding: 10px;
+}
+
+.social {
+    list-style: none;
+    display: inline-block;
+}
+
+.catalog-menu {
+    background: #141B22;
+}
+
+span {
```

Comment:
It’s not safe to assume that all span elements on the site will always need these styles.

Applying styles directly to the span tag can lead to unintended consequences because span is a very common element used throughout a site for inline text. Instead, it’s better to create a specific class for the styling you need and apply it to the relevant span elements. This approach gives you more control and prevents the accidental application of styles to span elements that shouldn’t have them.